src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -0,0 +1,3638 @@
+openapi: 3.0.1
+info:
```

Comment:
This is a huge file to look at! Maybe in the future, and if there's a way to do it, the yaml can be split into different files based on endpoints (using an include mechanism?) to make it easier to work on.